adafruit_usb_host_descriptors.py

```diff
@@ -70,3 +75,39 @@ def get_configuration_descriptor(device, index):
     full_buf = bytearray(wTotalLength)
     get_descriptor(device, DESC_CONFIGURATION, index, full_buf)
     return full_buf
+
+
+def find_mouse_endpoint(device):
```

Comment:
Should we leave room for finding other mouse endpoints?

```suggestion
def find_boot_mouse_endpoint(device):
```

Reply:
Yes, that is changed with the latest commit. Thank you.